src/shared/extensionGlobals.ts

```diff
@@ -86,6 +86,8 @@ export interface IconPaths {
     cloudWatchLogGroup: string
     bucket: string
     createBucket: string
+    renderStateMachineGraph: string
+    previewStateMachineCDK: string
```

Comment:
Are you still using both of these? If you consolidate to the `statemachine` icon, will you need either of these? Same goes for the other change in this file.